www/redirects.yaml

```
@@ -138,3 +138,7 @@
   # redirecting gatsby-source-filesystem Programmatic Import to plugin README
 - fromPath: /docs/gatsby-source-filesystem-programmatic-import/
   toPath: /packages/gatsby-source-filesystem/
+- fromPath: /docs/contributing/blog-and-website-contributions/
+  toPath: /blog-contributions/
+- fromPath: /docs/contributing/docs-and-website-components/
+  toPath: /docs-and-blog-components/
```

redirs should be full path...
```suggestion
- fromPath: /docs/contributing/blog-and-website-contributions/
  toPath: /docs/contributing/blog-contributions/
- fromPath: /docs/contributing/docs-and-website-components/
  toPath: /docs/contributing/docs-and-blog-components/
```